src/main/java/com/thebois/views/IProjector.java

```diff
@@ -0,0 +1,20 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.math.Vector2;
+
+/**
+ * Interface for a Projector.
+ */
+public interface IProjector {
+
+    /**
+     * Unprojects screen coordiantes to world coordinates.
+     *
+     * @param screenCordX X coordinate of the screen.
+     * @param screenCordY Y coordinate of the screen.
```

Comment:
These are better described as the x- and y-coordinates *of a position in screen space*. **Not** the position *of the screen*.